src/pages/login/Login.tsx

```diff
@@ -101,14 +103,18 @@ const Login = () => {
         <Grid container item justifyContent="center">
           <Grid item xs={6}>
             <Typography
-              variant="body2"
+              variant="body1"
               mb={7}
               color="textPrimary"
               sx={{
                 textAlign: 'center',
               }}
             >
-              {t('loginPage.description')}
```

Comment:
just comment this line referring to the issue

In general, let's mark each temporary change in order to simplify the revert of the issue